src/utils/sig_store.rs

```diff
@@ -359,6 +359,7 @@ impl<T: ZeroCopy + 'static> SigStore<T> {
                 .read(true)
                 .write(true)
                 .create(true)
+                .truncate(true)
```

Comment:
If we overwrite a file with a smaller one we should remove the extra bytes at the end